dodola/tests/test_services.py

```diff
@@ -443,12 +448,16 @@ def test_regrid_weights_integration(domain_file, tmpdir):
     assert actual_shape == expected_shape
 
 
-def test_clean_cmip6():
-    """Tests that cmip6 cleanup removes extra dimensions on dataset"""
+@pytest.mark.parametrize(
+    "gcm_variable", [pytest.param("tasmax"), pytest.param("tasmin"), pytest.param("pr")]
+)
+def test_clean_cmip6(gcm_variable):
+    """Tests that cmip6 cleanup removes extra dimensions on dataset
+    and that precip units are converted if variable is precip"""
```

Comment:
This "and that precip units are converted if variable is precip" should be a new test. You can cut-n-paste this test to easily create a new test, but this is testing a different feature/behavior so I'd make it a different test. 👍

Reply:
Good point - I've updated this.